packages/graphql-mesh-server/assets/handlers/invalidate-cloudfront-cache.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  CloudFrontClient,
+  CreateInvalidationCommand,
+  CreateInvalidationCommandInput,
+  CreateInvalidationCommandOutput,
+} from "@aws-sdk/client-cloudfront";
+
+const cfClient = new CloudFrontClient();
+
+const PATHS = process.env.PATHS as string;
+const DISTRIBUTION_ID = process.env.DISTRIBUTION_ID as string;
+
+export const handler = async (): Promise<CreateInvalidationCommandOutput> => {
+  const paths = PATHS.split(",");
+
+  const invalidationInput: CreateInvalidationCommandInput = {
+    DistributionId: DISTRIBUTION_ID,
+    InvalidationBatch: {
+      Paths: {
+        Quantity: paths.length,
+        Items: paths,
+      },
+      CallerReference: "lambda",
+    },
+  };
+
+  const command = new CreateInvalidationCommand(invalidationInput);
+  return await cfClient.send(command);
```

Comment:
If you're returning the result you don't need to await the result

Reply:
Oh really? I was worried if I didn't await the lambda might not finish sending the command

Reply:
Nah the function itself is async, so there is an implied await on the result (Could be wrong here but that's my understanding).